roles/openshift_logging/files/generate-jks.sh

```diff
@@ -73,6 +73,7 @@ function generate_JKS_chain() {
         -keystore $dir/$NODE_NAME.jks \
         -storepass $ks_pass \
         -noprompt \
+        -trustcacerts \
```

Comment:
@jcantrill Indeed it makes no difference, actually I should've added the flag to the command that adds the CA. I couldn't make it work running the deployer but I don't know what could be the reason.